cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -89,11 +90,11 @@ if [[ "${#buckets[@]}" -gt 0 ]]; then
   timeout 30m gsutil -m rm -r "${buckets[@]}"
 fi
 
-FAILED="$(ls "${ARTIFACTS}"/deprovision/*/failure)"
+FAILED="$(find ${clusters} -name failure -printf '%H\n')"
```

Comment:
What does the `%H` do? When I tried this out it always printed a literal dot newline

Reply:
```
%H     Starting-point under which file was found.
```

Reply:
Doesn't that mean this will always print the value of the `clusters` variable?

Reply:
`$clusters` is a list of directories (see line 72).  `find` will print the directory that originated the search that found each file named `failure`:

```
$ tree
.
└── deprovision
    ├── a
    │   └── failure
    ├── b
    ├── c
    │   └── failure
    └── d

5 directories, 2 files
$ find deprovision/* -name failure -printf '%H\n'
deprovision/a
deprovision/c
$ ARTIFACTS=$PWD ipi-deprovision/ipi-deprovision.sh
Deprovision failed on the following clusters:
a
c
```